### ISSUE TITLE
Animation state gets only updated when inside main camera

### ISSUE THREAD

**Issue by zzuegg** [jMonkeyEngine/jmonkeyengine#1875]
As the title says the animation state gets only updated when a geometry is inside the main camera. When animation is combined with shadow mapping it should also be updated in case any shadow cam is rendering the object. 
Here is a gist to test. 

https://gist.github.com/zzuegg/a1b10121bb49d983d36beb749e7f1dd0

**Comment by zzuegg** [comment]
I dont think it is culling related bat that the control does not get updated. but i habe to investigate. that part of the engine is new to me

**Comment by Ali-RS** [comment]
Just a note, afaik control `update` method is called always even if not inside the camera view, it is the control `render` method that is not called when outside of the camera.

**Comment by zzuegg** [comment]
Ok: ControlRender is only called when RenderManager.renderScene /renderSubScene is used.
